client/src/pages/index.js

```diff
@@ -1,20 +1,48 @@
 import React from "react";
-import Footer from "../components/Footer";
-import FileUploader from "../components/FileUploader";
+import "./index.css";
+import Footer from "../../src/components/Footer";
+import img1 from "../../src/images/img1.png";
+
+import Button from "../../src/components/Button";
```

Comment:
instead of Button and Button2 , rename to actual button names. like for login button rename to login